Header header

Wheel[] wheels

geometry_msgs/Pose2D robot_pose

geometry_msgs/Pose2D first_pose
geometry_msgs/Pose2D second_pose

float32 score
float32 orientation
float32 interval
float32 size
bool    ready
